distribution/ddf-common/src/main/resources/etc/system.properties

```diff
@@ -173,3 +173,8 @@ karaf.secured.services=(&(osgi.command.scope=*)(osgi.command.function=*))
 #org.osgi.framework.trust.repositories=${karaf.home}/etc/trustStore.ks
 ws-security.subject.cert.constraints = .*
 security.audit.roles=group,admin,manager,viewer,system-admin,system-history,systembundles
+
+#
+# Disable Hazelcast version check
+#
+hazelcast.version.check.enabled=false
```

Comment:
Looks like this property might be deprecated.  Not sure if it was in the version of Hazelcast we are using and what the right way is to disable version checking for newer versions.

https://github.com/hazelcast/hazelcast/blob/167361456fe0b8d77b57aae72dd415f8ade56f61/hazelcast/src/main/java/com/hazelcast/spi/properties/GroupProperty.java#L157-L159